packages/cmf/src/middlewares/http/middleware.js

```diff
@@ -162,7 +162,9 @@ export const httpMiddleware = ({ dispatch }) => next => action => {
 					} finally {
 						if (httpAction.onError) {
 							dispatch(onError(httpAction, errorObject));
-						} else {
+						}
+
+						if (typeof httpAction.onError !== 'function') {
```

Comment:
so it is dispatched with two different way ? PR description does not reflect that